ngraph/python/tests/test_ngraph/test_create_op.py

```diff
@@ -896,8 +896,8 @@ def test_prior_box_clustered(int_dtype, fp_dtype):
     image_size = np.array([64, 64], dtype=int_dtype)
     attributes = {
         "offset": fp_dtype(0.5),
-        "widths": np.array([4.0, 2.0, 3.2], dtype=fp_dtype),
-        "heights": np.array([1.0, 2.0, 1.0], dtype=fp_dtype),
+        "width": np.array([4.0, 2.0, 3.2], dtype=fp_dtype),
```

Comment:
this looks like not backward compatible change. If we want to make it then we need to introduce new version of the operation.

Reply:
So what can I do here, if _visit_attributes_ needs this to be stated explicitly if the parameter's name is height or heights, and the spec and IR in IE functional tests say this should be _width_ and _height_?